sandbox/change-sample-data/test-vsp-hmis-participating.R

```diff
@@ -0,0 +1,35 @@
+# run 00-Start_here.R 
+
+PDDEcols = c("OrganizationName",
+             "ProjectID",
+             "ProjectName",
+             "Issue",
+             "Type",
+             "Guidance",
+             "Detail")
+
+vsp_projects <- Project %>%
+  inner_join(Organization %>%
+              filter(VictimServiceProvider == 1),
+            by = "OrganizationID") %>%
+  pull(ProjectID) %>%
+  unique()
+  
+participating_projects <- Project %>%
+  inner_join(HMISParticipation %>%
+               filter(HMISParticipationType == 1),
+             by = "ProjectID") %>%
+  pull(ProjectID) %>%
+  unique()
+
+vsps_that_are_hmis_participating <- 
+  base::intersect(vsp_projects, participating_projects)
```

Comment:
this code^ is all repeat from the pdde_checker.R and is here for convenience only. the real value in this script is below (what we expect)